samza-api/src/main/java/org/apache/samza/table/descriptors/RemoteTableDescriptor.java

```diff
@@ -173,6 +179,41 @@ public RemoteTableDescriptor(String tableId) {
     return this;
   }
 
+  /**
+   * Disable both read and write rate limiter. If the read rate limiter is enabled, the user must provide a rate limiter
+   * by calling {@link #withRateLimiter(RateLimiter, TableRateLimiter.CreditFunction, TableRateLimiter.CreditFunction)}
+   * or {@link #withReadRateLimit(int)}. If the write rate limiter is enabled, the user must provide a rate limiter
+   * by calling {@link #withRateLimiter(RateLimiter, TableRateLimiter.CreditFunction, TableRateLimiter.CreditFunction)}
+   * or {@link #withWriteRateLimit(int)}. By default, both read and write rate limiters are enabled.
+   *
+   * @return this table descriptor instance.
+   */
+  public RemoteTableDescriptor<K, V> withDisableRateLimiter() {
```

Comment:
Can you explain what we are solving here? If the goal is to always provide a rate-limiter with remote-table,  maybe there's a simple way to get there.

For eg: we could mandate that users provide a readQPS/writeQPS or a rateLimiter when they construct a remote table. Users who prefer to avoid rate-limiting can set read/write QPS to infinity.

Reply:
@vjagadish1989 
Having a rate limier for a remote read/write table is highly recommended, and we will enable the rate limiter by default so that the user can either explicitly set a rate limiter or disable it. There is no change if the user is currently using rate limiter. Otherwise, the user either needs to set one or disable it using this API based on their use case. 
Ideally we should set a default one, but we can't get a correct estimation, and could result in problems, so we push the user to make the decision. 

This rate limiter is still an optional field. For optional field we don't put in the constructors.